packages/help-center/src/components/help-center-header.tsx

```diff
@@ -52,7 +52,7 @@ const HelpCenterHeader: React.FC< Header > = ( {
 	const [ chatWindowStatus, setChatWindowStatus ] = useState< WindowState >( 'closed' );
 	const [ unreadCount, setUnreadCount ] = useState< number >( 0 );
 	const formattedUnreadCount = unreadCount > 9 ? '9+' : unreadCount;
-	const popup = useSelect( ( select ) => select( STORE_KEY ).getPopup() );
+	const popup = useSelect( ( select ) => select( HELP_CENTER_STORE ).getPopup() );
```

Comment:
Could we also take the chance to remove the `popup`, as we are not using it anymore?